pulp_rpm/app/viewsets/acs.py

```diff
@@ -49,50 +49,51 @@ class RpmAlternateContentSourceViewSet(AlternateContentSourceViewSet, RolesMixin
                 "action": ["retrieve"],
                 "principal": "authenticated",
                 "effect": "allow",
-                "condition": "has_model_or_obj_perms:rpm.view_rpmalternatecontentsource",
+                "condition": "has_model_domain_or_obj_perms:rpm.view_rpmalternatecontentsource",
```

Comment:
These now have an additional 'or' in them.
```suggestion
                "condition": "has_model_or_domain_or_obj_perms:rpm.view_rpmalternatecontentsource",
```

Reply:
Ah! I missed these, thanks